base/c.jl

```diff
@@ -4,6 +4,11 @@
 
 import Core.Intrinsics: cglobal, bitcast
 
+# Equivalent to Sys.iswindows(), but that's not yet defined
+const _WINDOWS = let k = ccall(:jl_get_UNAME, Any, ())
```

Comment:
Is this a necessary change? Giving it a name at least avoids having to do the `ccall` several times, plus calling it Windows here is clearer than having to know that the Windows uname is NT. I don't care too much though, just seems unnecessary to change what's here.